client.go

```diff
@@ -134,6 +162,49 @@ func (c *Client) Browse(req *ua.BrowseRequest) (*ua.BrowseResponse, error) {
 	return res, err
 }
 
+// GetEndpoints reads all available endpoints from the server at endpoint
+func GetEndpoints(endpoint string) ([]*ua.EndpointDescription, error) {
```

Comment:
I'll refactor the client to split the `Open()` method to just connect with a secure channel and then to create a session separately. Then we can also add `GetEndpoints()` as a method to client and make this function a convenience helper.